Client/Frontend/Browser/BrowserViewController/BrowserViewController+Menu.swift

```diff
@@ -68,8 +68,13 @@ extension BrowserViewController {
                     }
                 }
             }
+            MenuItemButton(icon: #imageLiteral(resourceName: "settings-save-logins").template, title: Strings.passwordsMenuItem) {
+                let vc = LoginListViewController(profile: self.profile)
+                vc.settingsDelegate = self
+                menuController.pushInnerMenu(vc)
+            }
```

Comment:
Looks like this change snuck into this PR from login list PR

Reply:
Removed 
Fix at https://github.com/brave/brave-ios/pull/4220/commits/2c24efcf0732de14988fb61de174f8fe57b56614

Reply:
Indentation is fixed